src/main/java/ir/sahab/testcontainers/DnsAwareGenericContainer.java

```diff
@@ -60,6 +61,13 @@ public String getContainerIp() {
         return getContainerIp(getContainerInfo());
     }
 
+    /**
```

Comment:
This below methods retrieve container (anything) and returns its IP whether this container is 'DnsAwareGenericContainer' or something else. Shouldn't we place them in utility class like 'ContainerInfoUtility' or 'ContainerNetworkUtility' (Maybe we change it to ContainerUtility in future but it is better to be specific for now. 

What is your  opinion?

Reply:
Done